### PR TITLE
Feat: Add support for MotherDuck

### PR DESCRIPTION
**Pull request by izeigerman**

### CODE REVIEW

File: docs/integrations/overview.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ SQLMesh supports the following execution engines for running SQLMesh projects:
 * [BigQuery](./engines/bigquery.md)
 * [Databricks](./engines/databricks.md)
 * [DuckDB](./engines/duckdb.md)
+* [DuckDB](./engines/motherduck.md)
```
Comment (eakmanrq) [inline docs/integrations/overview.md:17]: Typo: Motherduck

**Review by eakmanrq — COMMENTED** [review]
Does Motherduck not support multiple catalogs or are we just not supported multiple catalogs at first?

**Review by tobymao — APPROVED** [review]

**Review by eakmanrq — COMMENTED** [review]
Can you add this as a remote connection in the test integration file?

**Comment by izeigerman** [comment]
@eakmanrq I'm not quite sure how multi-catalog works with MotherDuck, so I'd rather defer implementation until there's actual demand for it.

**Review by eakmanrq — APPROVED** [review]

**Pull request merged**